src/shared/Core/Trace2.cs

```diff
@@ -69,6 +70,22 @@ public interface ITrace2 : IDisposable
         double timeInMilliseconds,
         [System.Runtime.CompilerServices.CallerFilePath] string filePath = "",
         [System.Runtime.CompilerServices.CallerLineNumber] int lineNumber = 0);
+    
+    /// <summary>
+    /// Writes exit code and execution time to trace writer.
+    /// </summary>
+    /// <param name="code">Application exit code.</param>
+    /// <param name="timeInMilliseconds">Elapsed time in milliseconds since GCM started.</param>
+    /// <param name="filePath">Path of the file this method is called from.</param>
+    /// <param name="lineNumber">Line number of file this method is called from.</param>
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1026:DefaultParametersShouldNotBeUsed")]
+    void WriteExit(
```

Comment:
Are we calling this from anywhere? Do we have an existing handler for process termination?

Reply:
I suppose it could be handled within the `Dispose()` method of the `Trace2` class, but that's probably not the best place.

Reply:
Sigh. I took a stab at implementing this (with a comment along the lines of `// TODO: Determine whether this is the right place to record this.`) but it appears to have fallen off during one of the many rebases I did to try and get this PR in shape. Thanks for pointing this out!